BenchmarkTests/README.md

```diff
@@ -2,6 +2,16 @@
 
 [Synthetics for Mobile](https://docs.datadoghq.com/mobile_app_testing/) runs Benchmark test scenarios to collect metrics of the SDK performances.
 
+## Setup
+
+To open the project with the correct environment, run the following command:
+
+```bash
+make benchmark-tests-open
+```
+
+> [!TIP]
+> If XCode is failing to install packages with a message such as **failed to clone**, **requires authentification** or **dependency failed**, it might be due to a Git configuration that forces HTTPS URLs to be rewritten as SSH. In that case, try commenting it out in your global Git config (~/.gitconfig) and fetching the packages again.
```

Comment:
```suggestion
> If Xcode is failing to install packages with a message such as **failed to clone**, **requires authentification** or **dependency failed**, it might be due to a Git configuration that forces HTTPS URLs to be rewritten as SSH. In that case, try commenting it out in your global Git config (~/.gitconfig) and fetching the packages again.
```